SYCL/InlineAsm/asm_float_imm_arg_aspect-fp64.cpp

```diff
@@ -0,0 +1,11 @@
+// Enable FP64 part of <asm_float_imm_arg.cpp>. To be removed once DPC++
+// supports optional device features and the code could be enabled
+// unconditionally without causing failures in speculative compilation
+// of the kernels.
```

Comment:
Same as above.

Reply:
Fixed in `asm_float_imm_arg.cpp` by using function template.